Bio/codonalign/__init__.py

```diff
@@ -36,7 +36,6 @@ def build(
     gap_char="-",
     unknown="X",
     codon_table=default_codon_table,
```

Comment:
Better not to have a mutable object as a default argument, how about ``codon_table=None`` and then:

```python
if not codon_table:
    codon_table= default_codon_table
```

(Yes, this was a pre-existing issue - but see also my query about taking a deepcopy for the default)

Reply:
OK